matRad/dicom/matRad_interpDicomCtCube.m

```diff
@@ -1,16 +1,16 @@
-function interpCt = matRad_interpDicomCtCube(origCt, origCtInfo, resolution, grid)
+function interpCt = matRad_interpDicomCtCube(origCt, origCtInfo, resolution, doseGrid)
 % matRad function to interpolate a 3D ct cube to a different resolution
 %
 % call
 %   interpCt = matRad_interpDicomCtCube(origCt, origCtInfo, resolution)
-%   interpCt = matRad_interpDicomCtCube(origCt, origCtInfo, resolution, grid)
+%   interpCt = matRad_interpDicomCtCube(origCt, origCtInfo, resolution, doseGrid)
 %
 % input
 %   origCt:         original CT as matlab 3D array
 %   origCtInfo:     meta information about the geometry of the orgiCt cube
 %   resolution:     target resolution [mm] in x, y, an z direction for the
 %                   new cube
-%   grid:           optional: externally specified grid vector
+%   doseGrid:       optional: externally specified grid vector
```

Comment:
maybe change from 'doseGrid' to 'importGrid'

Reply:
Was changed in this update https://github.com/e0404/matRad/pull/758/commits/c5b44adb21f8a5a279db925d26e087b3b0749f0f